src/libraries/System.Private.Xml/src/System/Xml/Core/XmlRawWriter.cs

```diff
@@ -274,7 +274,7 @@ internal virtual void WriteQualifiedName(string prefix, string localName, string
             if (prefix.Length != 0)
             {
                 WriteString(prefix);
-                WriteString(":");
+                WriteCharEntity(':');
```

Comment:
Why would you want to create a character entity here? (and in the other place that you modified)